js/views/modals/wallet/Transaction.js

```diff
@@ -0,0 +1,198 @@
+      this.setState({
+        retryConfirmOn: false,
+      });
+    }
+  }
+
+  onClickRetryConfirmed() {
+    this.setState({
+      retryInProgress: true,
+      retryConfirmOn: false,
+    });
+
+    this.retryPost = $.post(app.getServerUrl(`wallet/bumpfee/${this.model.id}`))
+      .always(() => {
+        this.setState({
+          retryInProgress: false,
+        });
+      }).fail((xhr) => {
+        if (xhr.statusText === 'abort') return;
+        const failReason = xhr.responseJSON && xhr.responseJSON.reason || '';
+        openSimpleMessage(
+          app.polyglot.t('wallet.transactions.transaction.retryFailDialogTitle'),
+          failReason);
+      })
+      .done(data => {
+        this.trigger('retrySuccess', { data });
+        this.model.set('canBumpFee', false);
+      });
+  }
+
+  onClickRetryPmt() {
+    if (this.getFeeLevel) this.getFeeLevel.abort();
+    this.getFeeLevel = this.options.getFeeLevel('PRIORITY');
+
+    const state = {
+      retryConfirmOn: true,
+    };
+
+    if (this.getFeeLevel.state() === 'pending') {
+      this.setState({
+        ...state,
+        fetchingEstimatedFee: true,
+      });
+    }
+
+    this.getFeeLevel.done(fee => {
+      this.setState({
+        ...state,
+        fetchingEstimatedFee: false,
+        estimatedFee: this.feeToBtc(fee),
+      });
+    });
+
+    // don't bubble to the document click handler
+    return false;
+  }
+
+  onClickRetryConfirmCancel() {
+    this.closeRetryConfirmBox();
+  }
+
+  onClickTxidLink(e) {
+    this.setState({
+      copiedIndicatorOn: true,
+    });
+
+    clipboard.writeText($(e.target).text());
+    clearTimeout(this.copiedIndicatorTimeout);
+
+    this.copiedIndicatorTimeout = setTimeout(() => {
+      this.setState({
+        copiedIndicatorOn: false,
+      });
+    }, 1000);
+  }
+
+  getState() {
+    return this._state;
+  }
+
+  setState(state, replace = false) {
+    let newState;
+
+    if (replace) {
+      this._state = {};
+    } else {
+      newState = _.extend({}, this._state, state);
+    }
+
+    if (!_.isEqual(this._state, newState)) {
+      this._state = newState;
+      this.render();
+    }
+
+    return this;
+  }
+
+  feeToBtc(fee) {
+    // We'll approximateally match the server's algorythm to estimate the fee.
+    // Fee is per byte in satoshi. A estimated average transaction is 200 bytes.
+    // So we'll multiply the fee by 200, divide by a 100 mil to get BTC and
+    // then multiply by 2 (the server bumps the fee by doubling it.)
+    return fee * 200 / 100000000 * 2;
+  }
+
+  closeRetryConfirmBox() {
+    if (this.getFeeLevel) this.getFeeLevel.abort();
+    this.setState({
+      retryConfirmOn: false,
+      fetchingEstimatedFee: false,
+    });
+  }
+
+  get $retryPmtConfirmedBox() {
+    return this._$retryPmtConfirmed ||
+      (this._$retryPmtConfirmed = this.$('.js-retryPmtConfirmed'));
+  }
+
+  remove() {
+    if (this.retryPost) this.retryPost.abort();
```

Comment:
They're aborted in the remove() of the Transactions (plural) view, which is where they originate from. I don't want the child Transaction views to abort them because, other views could theoretically be waiting on the same calls, since they are cached and potentially shared by different views.